### ISSUE TITLE
[Important accessibility improvement] Emphasize current active main navigation item/route

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#2626]
[This is a ticket for the "accessibility backlog"]

<img width="317" alt="Screenshot of phoenix's main menu. The current active routes font color, All files, is a little bit darker than the other navigation items" src="https://user-images.githubusercontent.com/505181/70323842-3382e500-182e-11ea-9196-cd0441fc8f20.png">

# Visually
As of now, the contrast between normal navigation points and the active one (in the image above, "Alle Dateien") is to low

# Non-visually
The currently active navigation item should announce this state to screen readers using `aria-active="page"`

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
